paper/paper.bib

```diff
@@ -0,0 +1,133 @@
+@Article{Seurat,
+  author = {Tim Stuart and Andrew Butler and Paul Hoffman and
+    Christoph Hafemeister and Efthymia Papalexi and William M Mauck
+    III and Yuhan Hao and Marlon Stoeckius and Peter Smibert and
+    Rahul Satija},
```

Comment:
wait, they just do and and and and and, and no commas?

Reply:
yup: https://texfaq.org/FAQ-manyauthor

Reply:
I'd hate to be the person whose name is `and`.